File: kore_api/kore_api/koredb/Organization.cs
using System;
using System.Collections.Generic;

namespace kore_api.koredb
{
    public partial class Organization
    {
        public Organization()
        {
            Account = new HashSet<Account>();
            Task = new HashSet<Task>();
            Taskdepartment = new HashSet<Taskdepartment>();
            Taskmembership = new HashSet<Taskmembership>();
            Tasktype = new HashSet<Tasktype>();
        }

        public int Id { get; set; }
        public string Name { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime DateModified { get; set; }
        public int? Status { get; set; }
        public int? CreatedBy { get; set; }
        public int? ModifiedBy { get; set; }

        public ICollection<Account> Account { get; set; }
        public ICollection<Task> Task { get; set; }
        public ICollection<Taskdepartment> Taskdepartment { get; set; }
        public ICollection<Taskmembership> Taskmembership { get; set; }
        public ICollection<Tasktype> Tasktype { get; set; }
    }
}

File: kore_app/lib/models/organization.dart

class Organization {
  int id;
  String name;
  DateTime dateCreated;
  DateTime dateModified;
  int status;
  int createdBy;
  int modifiedBy;

  Organization({
    this.id,
    this.name,
    this.dateCreated,
    this.dateModified,
    this.status,
    this.createdBy,
    this.modifiedBy
  });

  factory Organization.fromJson(Map<String, dynamic> json ) {
    return new Organization(
        id: json['id'] as int,
        name: json['name'] as String,
        dateCreated: json['dateCreated'] as DateTime,
        dateModified: json['dateModified'] as DateTime,
        status: json['status'] as int,
        createdBy: json['createdBy'] as int,
        modifiedBy: json['modifiedBy'] as int,
    );
  }
}